frontend/src/components/Cart/CartSidebarItem.vue

```diff
@@ -19,8 +19,36 @@
 				font="text-md"
 			/>
 
-			<div class="flex flex-col justify-between w-full">
+			<div class="flex flex-col justify-between w-full relative">
 				<div class="text-sm font-semibold">{{ row.item_name }}</div>
+				<!-- Delete Menu Only Desktop Screens -->
+				<div class="cursor-pointer absolute -top-1 -right-1">
```

Comment:
Don't see the need for cursor-pointer since we're adding a button inside

Reply:
Oops! Remove karna bhool gaya..